### PR TITLE
Feat: implement get_execute_order_params

### PR DESCRIPTION
**Pull request by dbejarano820**
# Pull Request type
Please add the labels corresponding to the type of changes your PR introduces:
- Feature

## What is the current behavior?
Resolves: #356 

## What is the new behavior?
Implemented the get_execute_order_params which is the only function in the contract

## Does this introduce a breaking change?
No

Clearly there are several util functions that are used that are yet implemented.

### CODE REVIEW

File: src/exchange/base_order_handler.cairo [hunk 1]
```diff
@@ -186,55 +187,42 @@ mod BaseOrderHandler {
...
-            };
+
+            let min_oracle_block_numbers = get_uncompacted_oracle_block_numbers(
+                oracle_params.compacted_min_oracle_block_numbers.span(), oracle_params.tokens.len()
+            );
+            let max_oracle_block_numbers = get_uncompacted_oracle_block_numbers(
+                oracle_params.compacted_max_oracle_block_numbers.span(), oracle_params.tokens.len()
+            );
+
+            let market = market_utils::get_enabled_market(data_store, order.market);
```
Comment (dbejarano820) [inline src/exchange/base_order_handler.cairo:213]: is it ok if we don't check if `order.market != zero.address` before doing this?

**Comment by dbejarano820** [comment]
What scenarios should I be testing? Would the missing util implementations influence the tests carried out?

File: src/exchange/base_order_handler.cairo (hunk 1 above)
Comment (zarboq) [reply src/exchange/base_order_handler.cairo:213]: Better to check else I think it will revert inside `get_enabled_market`

**Comment by zarboq** [comment]
Mmh since testing would just consist of asserting that returned fields are ok and knowing that most of the called functions called in `get_execute_order_params` are missing I think it doesn't make much sense to test it right now. But it's good that you created the test file, just add a TODO to test when all functions will be implemented and it should be ok

File: src/exchange/base_order_handler.cairo (hunk 1 above)
Comment (dbejarano820) [reply src/exchange/base_order_handler.cairo]: will do!

File: src/exchange/base_order_handler.cairo [hunk 2]
```diff
@@ -186,55 +187,53 @@ mod BaseOrderHandler {
...
+            let min_oracle_block_numbers = get_uncompacted_oracle_block_numbers(
+                oracle_params.compacted_min_oracle_block_numbers.span(), oracle_params.tokens.len()
+            );
+            let max_oracle_block_numbers = get_uncompacted_oracle_block_numbers(
+                oracle_params.compacted_max_oracle_block_numbers.span(), oracle_params.tokens.len()
+            );
+
+            let address_zero = 0.try_into().unwrap();
+
+            let mut market = Market {
```
Comment (zarboq) [inline src/exchange/base_order_handler.cairo:215]: we can use default here

**Review by zarboq — CHANGES_REQUESTED** [review]
Small change then good to go :fire:

File: src/exchange/base_order_handler.cairo (hunk 2 above)
Comment (dbejarano820) [reply src/exchange/base_order_handler.cairo]: done

**Review by zarboq — APPROVED** [review]

**Pull request merged**